modules/ROOT/pages/about-classloading-isolation.adoc

```diff
@@ -124,7 +124,9 @@ See xref:mmp-concept.adoc#configure-shared-libraries[Configure Shared Libraries]
 
 Shared libraries are visible to all connectors in the application. Configuring shared libraries can cause issues if the shared library has different versions of the same classes that exist in other connectors or their dependencies.
 
-In case you need to configure a different version of the same dependency for different connectors, configure them as plugin dependencies. Plugin dependencies are configured at connector level and are visible only to that connector.
+To avoid such conflicts, you can configure additional plugin dependencies to enrich a connector's classpath. This configuration enables you to effectively add new dependencies to a connector, as if the dependencies were declared in the connector's `pom.xml` file. +
```

Comment:
I still feel that we need to somehow state that this is a different approach to solve the same problem. You can use either <sharedLibraries> or <additionalPluginDepenndencies> but, you will rarely use both. 
That's why i would add something like: There is another configuration option....

Reply:
Do you think it is better to have a separate section to define additional plugin dependencies? Please check commit c3c4b1b and let me know if that's ok.

Reply:
If we add a different section, i would arrange everything differently. First, showing the problem we have to solve (add dependencies to connectors) and then show the 2 solutions.

Reply:
I created an intro to the topic, and then presented the two solutions. 
Please check 9691c12 and let me know if the introductory section is accurate.